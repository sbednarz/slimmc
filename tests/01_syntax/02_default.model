list parameters
0:print

