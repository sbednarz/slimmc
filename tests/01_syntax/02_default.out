[slimmc]
processing model file: 02_default.model
simulation parameters:
kd=0.0 1/s
f=0.0
ki=0.0 L/(mol*s)
kp=0.0 L/(mol*s)
ktc=0.0 L/(mol*s)
ktd=0.0 L/(mol*s)
cI0=0.0 mol/L
cM0=0.0 mol/L
cRx0=0.0 mol/L
cPx0=0.0 mol/L
cD0=0.0 mol/L
MwM=0.0 g/mol
V_MC=1e-17 L
seed=random
simulation initialized
simulation time from 0 to 0.0s, 1 of breakpoints
